lib/store_finder/haversine.ex

```diff
@@ -18,4 +8,14 @@ defmodule StoreFinder.Haversine do
         else: acc
     end, [], :store_cache)
   end
+
+  defp calc_distance({lat1, long1}, {lat2, long2}) do
+    v = :math.pi / 180
```

Comment:
please consider expanding the variable names for clarity (_or adding in-line comments_).